packages/@webex/plugin-meetings/src/interceptors/locusRouteToken.ts

```diff
@@ -0,0 +1,80 @@
+/*!
+ * Copyright (c) 2015-2020 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import {Interceptor} from '@webex/http-core';
+import {has} from 'lodash';
+
+const X_CISCO_PART_ROUTE_TOKEN = 'X-Cisco-Part-Route-Token';
+const ROUTE_TOKEN = {};
+
+/**
+ * @class LocusRouteTokenInterceptor
+ */
+export default class LocusRouteTokenInterceptor extends Interceptor {
+  /**
+   * @returns {LocusRouteTokenInterceptor}
+   */
+  static create() {
+    // @ts-ignore
+    return new LocusRouteTokenInterceptor({webex: this});
+  }
+
+  getActiveMeetingId() {
+    // @ts-ignore
+    return this.webex.meetings.meetingCollection.getActiveWebrtcMeeting()?.id;
```

Comment:
we can't use getActiveWebrtcMeeting() here. There may be users who join a meeting without having a webrtc connection (if they just join, but don't add media) and also we we may need to get and pass that header before the webrtc connection is created. 
Ideally we would want to use the meeting object for which that request was sent out, but I guess you have no way of getting that from the request itself, so probably the 2nd best option is to use the request url and extract the locus url that's specific to that user in that meeting from it